grammar/test/build.sh

```diff
@@ -0,0 +1,3 @@
+java -Xmx500M -cp "/usr/local/lib/antlr-4.7.1-complete.jar:$CLASSPATH" org.antlr.v4.Tool FloatingPointExpression.g4 
```

Comment:
`/usr/local/XXX` requires root access by default, so I think that we should move the reference. Nitpicky, but just my opinion

Reply:
:+1: